src/index.js

```diff
@@ -1,16 +1,22 @@
 import {dispatch} from './lib/utils.js'
 import ConfirmationController from './lib/ConfirmationController.js'
 
-const confirm = (_message, _formElement, submitter) => {
-  const confirmationResponse = dispatch('confirm', submitter)
+const confirm = async (message, _formElement, submitter) => {
```

Comment:
does this not represent a change in the public API?

Reply:
@seanpk yes, technically.  Though this method was never advertised as anything except for what you should feed to `Turbo.setConfirmMethod`.  To use it directly, you would have had to provide an element with a `data-turbo-confirm` attribute as the third argument.  That will continue to work with the new API.  I'm only attempting to make that use-case much less of a hassle, by removing the element requirement and respecting a string message if supplied.

However, I do think this potentially warrants a major version bump.  If that's what you're asking.

Reply:
@seanpk @OutlawAndy As for me, I'd think that it's a major version bump. Otherwise, I right like it. Have we tested any of this yet? If we have, I'd love to see this get pushed and used!

Reply:
yes, a major version bump is what I was thinking